metadata-service/services/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -297,4 +298,11 @@ Pair<Boolean, List<Pair<String, RecordTemplate>>> generateDefaultAspectsOnFirstW
    */
   @Nonnull
   BrowsePathsV2 buildDefaultBrowsePathV2(final @Nonnull Urn urn, boolean useContainerPaths) throws URISyntaxException;
+
+  /**
```

Comment:
This does not seem good. 

Entity Client should wrap entity service....
I don't quite understand why this is happening here

Reply:
I think this is just a lack of understanding for what's happening inside of entity service